inlong-sort/sort-connectors/mongodb-cdc/pom.xml

```diff
@@ -74,6 +105,7 @@
                                     <include>org.apache.avro:avro</include>
                                     <include>org.apache.kafka:*</include>
                                     <include>com.fasterxml.*:*</include>
+                                    <include>com.google.code.gson:*</include>
```

Comment:
no need it

Reply:
MongoDBConnectorDeserializationSchema.java L720 need